src/strategies/bearer.strategy.js

```diff
@@ -22,7 +22,7 @@ module.exports.strategy = new Strategy(
     // available to the services
     if (!env.INTROSPECTION_URL) {
       let context = {token};
-      let scope = 'patient/*.read';
+      let scope = 'patient/*.read, procedure/*.read, condition/*.read, organization/*.read';
```

Comment:
I'm not sure these are valid scopes "procedure/*.read, condition/*.read, organization/*.read"  Generally the scopes take the form of patient/<resource_type>.<access>  so patient/Observation.read or patient/Condition.write.  User scopes have do the same but are prefixed with user/ instead of patient/